students/16th/wanie-kim/styles/main.css

```
@@ -0,0 +1,150 @@
+* {
+    box-sizing: border-box;
+}
+ul {
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    width: 100vw;
+    height: 100vh;
+    margin: 0px;
+}
+header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+}
+nav {
+    width: 100%;
+    height: 54px;
+    display: flex;
+    justify-content: center;
+    border-bottom: 1px solid #bbb;
+}
+
+.nav-main {
+    width: 975px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 20px;
+}
+.logo {
+    width: 360px;
+    height: 36px;
+    display: flex;
+    align-items: flex-end;
+}
+
+.nav-input {
+    width: 215px;
+    height: 28px;
+}
+
+input {
+    width: 100%;
+    height: 100%;
+    border: 1px solid #bbb;
+    background-color: #FAFAFA;
+    border-radius: 3px;
+    padding: 7px;
+}
+.logo img {
+    width: 103px;
+}
+.nav-right {
+    width: 360px;
+    height: 24px;
+    display: flex;
+    justify-content:flex-end;
+    list-style: none;
+}
+
+.nav-right li img {
```

클래스명이 nav-right태그 안에 있는 img태그를 선택하는 것이기 때문에 굳이 li태그를 작성하지 않고 아래와 같이 작성해도 되겠네요! 
```suggestion
.nav-right img {
```